src/components/Tabs/Tabs.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+    import { writable } from "svelte/store";
+    import TabsHead from "./TabsHead.svelte";
+    import TabItem from "./TabItem.svelte";
+
+    type Item = {
+        label: string;
+        href: string;
+        onClick?: () => void;
+    };
+
+    export let defaultActive: string = "0";
+    export let skipBorder: boolean = false;
+    export let items: Item[];
+
+    let className: string = "";
+    export { className as class };
+
+    let tab = writable<string>(defaultActive);
+    $: $tab = defaultActive;
+
+    let handleTabClick = (index: number, clbk: any) => {
+        tab.set(index.toString());
+        if (clbk) clbk();
+    };
+</script>
+
+<div class="tabs {className}">
+    <TabsHead skipBorder={skipBorder}>
```

Comment:
@tsejerome why do we need TabsHead? looks like overhead to me.
If there is head, then there is a body? But I can see only Head, which looks like a decorative wrapper around. Or does it have a functional purpose as well?